app.js

```diff
@@ -32,6 +32,8 @@ app.use(cookieParser());
 
 app.enable("trust proxy");
 
+app.use(express.static(__dirname + "/bower_components"));
+
```

Comment:
There's already a static `public` directory defined on line 399. I would remove these lines and place `bower_components` within the `public` directory (leaving room for other static files other than just Bower components). With this suggested approach, I would probably not name the directory `bower_components`, but something more generic such as `assets`. Alternatively, we could use an asset pipeline of some sort and place components more intentionally (e.g. `css`, `js`).